serial_windows.go

```diff
@@ -295,6 +323,15 @@ func (port *windowsPort) GetModemStatusBits() (*ModemStatusBits, error) {
 	}, nil
 }
 
+//sys createEvent(eventAttributes *uint32, manualReset bool, initialState bool, name *uint16) (handle syscall.Handle, err error) = CreateEventW
```

Comment:
Those are parsed by mksyscall to create the `syscall_windows.go` file.
Actually I just run `go generate` that use the command line [here](https://github.com/bugst/go-serial/pull/16/files/467fab473a66a6f5e94d75e050cfaa89441b142e#diff-13e062b00de9cb256140b6922b645aafR26) to invoke `mksyscall_windows`.